iml-system-docker-tests/tests/docker_integration_test.rs

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) 2020 DDN. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+use iml_system_test_utils::{docker, get_local_server_names, iml, vagrant};
+use std::{
+    collections::{hash_map::RandomState, HashMap},
+    time::Duration,
+};
+use tokio::time::delay_for;
+
+async fn setup() -> Result<(), Box<dyn std::error::Error>> {
+    // remove the stack if it is running and clean up volumes and network
+    docker::remove_iml_stack().await?;
+    docker::system_prune().await?;
+    docker::volume_prune().await?;
+    docker::configure_docker_overrides().await?;
+    docker::stop_swarm().await?;
+    iml_systemd::restart_unit("docker.service".into()).await?;
```

Comment:
Why restart here?

Reply:
After cleaning up the swarm, volumes, etc, I figured this would be a good place to restart docker in case there are any network hiccups like we've seen in the past. It's not entirely necessary so I can remove it if you think it's not necessary, but it just ensures the docker service is clean before we start.